src/components/ProductCard.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+export default function ProductCard({name, price, img, description, id, key,addToCart, formatPrice}) {
+    const product = {name, price, id,}
```

Comment:
I think this single syntax error here is causing you a whole lot of pain!

2 things:

1) You are already destructuring the parameters by including `{}` in the function definition, so you can delete this line.
2) If you wanted to do it the other way, the syntax looks like this:

```js
export default function ProductCard(props) {
    const {name, price, img, description, id, key,addToCart, formatPrice} = props
}
```